galata/src/helpers/performance.ts

```diff
@@ -40,6 +40,22 @@ export class PerformanceHelper {
     return time;
   }
 
+  /**
+   * Throttle network
+   */
+  async throttleNetwork(config: {
+    downloadThroughput: number;
+    uploadThroughput: number;
+    latency: number;
+  }): Promise<CDPSession> {
+    const cdpSession = await this.page.context().newCDPSession(this.page);
+    await cdpSession.send('Network.emulateNetworkConditions', {
+      offline: false,
+      ...config
+    });
+    return cdpSession;
```

Comment:
What about returning `null` in case the browser does not support CDP? As this may hit maintainer in the future if they turn on another browser.

```suggestion
  }): Promise<CDPSession | null> {
    const browserName = this.page.context().browser().browserType().name();
    if (browserName === 'chromium') {
      const cdpSession = await this.page.context().newCDPSession(this.page);
      await cdpSession.send('Network.emulateNetworkConditions', {
        offline: false,
        ...config
      });
      return cdpSession;
    }
    console.log('Browser does not support throttling network');
    return null;
```

Reply:
Thanks!

I was quite confused by the docs saying it is chromium only but then not describing whether it will throw, or work as a no-op and forget to tackle it in the end. Your solution works great for now.

Reply:
I agree the doc is not clear about what to expect for non-chromium browser.

Thanks for applying the suggestion